google/cloud/spanner_dbapi/exceptions.py

```diff
@@ -92,3 +92,13 @@ class NotSupportedError(DatabaseError):
     """
 
     pass
+
+
+class AbortedRetried(OperationalError):
```

Comment:
This should rather be called something like `RetryAborted`.

Reply:
Done